### PR TITLE
gopass: update to 1.12.4

### PR DESCRIPTION
**Pull request by dominikschulz**
The shell completion files are now part of the source tarball and doesn't need to be generated.

#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/gopass/template [hunk 1]
```diff
@@ -14,14 +14,7 @@ license="MIT"
...
 changelog="https://raw.githubusercontent.com/gopasspw/gopass/master/CHANGELOG.md"
 distfiles="https://github.com/gopasspw/gopass/archive/v${version}.tar.gz"
-checksum=73b7c5c8367e664d85871fba88f4fa806ef0fb75047c767a72b8516dea6f64b3
-
-post_build() {
-	gopass=$(find $GOPATH/bin -name gopass)
-	for shell in bash fish zsh; do
-		vtargetrun $gopass completion $shell > completion.$shell
-	done
-}
```
Comment (ericonr) [inline srcpkgs/gopass/template:24]: Why did you remove this? The `vcompletion` calls below will fail because they can't find the file...

Comment (dominikschulz) [reply srcpkgs/gopass/template:24]: Generating the completion files during package build time cause trouble for some platforms so we decided to generate them when cutting a release and include them in the source package, see https://github.com/gopasspw/gopass/commit/2fdeb0fd42801026afe178a75ce68d0df29de9f7

But I didn't notice the slightly different naming of these files. Should be fixed now.

Comment (ericonr) [reply srcpkgs/gopass/template:24]: In that case, please also `vman gopass.1` in `post_install`.

Comment (dominikschulz) [reply srcpkgs/gopass/template:24]: Indeed, good idea. Thanks.
